docs/unique/chroma-distance-metric-fix-3.md

```
@@ -528,3 +476,52 @@ It seemed to work this time
 [04-relations] refsForDoc ffb9b2a9-744d-4a53-9565-130fceae0832 chunks 90 refs 414
 [04-relations] refsForDoc fe7193a2-a5f7-4b3c-bea0-bd028815fc2c chunks 308 refs 1162
 [04-relations] refsForDoc f2d83a77-7f86-4c56-8538-1350167a0c6c chunks 146 refs 635
+<!-- GENERATED-SECTIONS:DO-NOT-EDIT-BELOW -->
+## Related content
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)